docs/api-types/volumesnapshotlocation.md

```diff
@@ -0,0 +1,56 @@
+# Ark Volume Snapshot Location
+
+## Volume Snapshot Location
+
+Ark can store volume snapshots in multiple persistent volume types in a single cluster. These are represented in the cluster via the `VolumeSnapshotLocation` CRD.
```

Comment:
I would make sure this sentence/section makes clear that (a) Ark can take snapshots of volumes from multiple providers, and (b) Ark can store snapshots in multiple locations per provider (e.g. for AWS, us-east-1, us-west-1, ...).  And a `VolumeSnapshotLocation` describes a provider + location